code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -55,6 +55,13 @@
 
 	return null
 
+/mob/living/carbon/is_ears_covered()
```

Comment:
I don't like this proc being named as if it returns a boolean when it actually returns an `atom | null`, I know we set that precedent with the `is_antagonist` macros but it makes me sad.

Reply:
it's named after / follows the precedent of the other procs `is_eyes_covered` and `is_mouth_covered`

Reply:
sad